client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -52,11 +54,11 @@ function createQuestionEpic(action$, state$, { window }) {
         `**Tell us what's happening:**
         \n\n
         ${
-          projectFormValues
+          projectFormValues.length > 0
```

Comment:
The following should suffice?
```suggestion
          projectFormValues.length
```